src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -892,6 +893,23 @@ public static ItemStack harvestPlant(Block block) {
 		return null;
 	}
 
+	public static void harvestFruit(Block fruit) {
+		Location loc = fruit.getLocation();
+		SlimefunItem check = BlockStorage.check(loc);
+		if (check == null) return;
+
+		for (Tree tree : ExoticGarden.getTrees()) {
+			if (check.getID().equalsIgnoreCase(tree.getFruitID())) {
```

Comment:
This should REEEAAAALLLY be a HashMap lookup tbh.

Reply:
Same as above, your code not mine ahaha Define HashMap lookup, do you want me to save a HashMap with all fruit IDs somewhere on enable and check against that?

Reply:
Yes